web/pdf_print_service.js

```diff
@@ -200,7 +211,17 @@
       if (!activeService) {
         console.error('Expected print service to be initialized.');
       }
-      activeService.renderPages().then(startPrint, abort);
+      var activeServiceOnEntry = activeService;
+      activeService.renderPages().then(function() {
+        activeServiceOnEntry.throwIfInactive();
+        return startPrint(activeServiceOnEntry);
+      }).catch(function() {
+        // Ignore any error messages.
+      }).then(function() {
+        if (activeServiceOnEntry.active) {
+          abort();
```

Comment:
`startPrint` starts the actual printing, so yes, it is also `performPrint`, or `startPrintForReal`. I will add a comment that explains the check.